docs/tutorial/part-eight/index.md

```diff
@@ -13,51 +13,197 @@ Wow! You've come a long way! You've learned how to:
+
+### ✋ Using `gatsby-plugin-offline`
+
+1. Install the plugin:
+
+```bash
+npm install --save gatsby-plugin-offline
+```
+
+2. Add the plugin to the `plugins` array in your `gatsby-config.js` file.
+
+```javascript
+{
+    plugins: [
+        {
+            resolve: `gatsby-plugin-manifest`,
+            options: {
+                ...
+            }
+        },
+        'gatsby-plugin-offline'
+    ]
+}
+```
+
+That's all you need to get started with service workers with Gatsby.
+
+> 💡 The manifest plugin should be listed *before* the offline plugin so that the offline plugin can cache the created `manifest.webmanifest`.
+
+## Add page metadata
+
+Adding metadata to pages (such as a title or description) are key in helping search engines like Google understand your content, and decide when to surface it in search results.
+
+[React Helmet](https://github.com/nfl/react-helmet) is a package that provides a React component interface for you to manage your [document head](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/head).
+
+Gatsby's [react helmet plugin](/packages/gatsby-plugin-react-helmet/) provides drop-in support for server rendering data added with React Helmet. Using the plugin, attributes you add to React Helmet will be added to the static HTML pages that Gatsby builds.
+
+### ✋ Using `React Helmet` and `gatsby-plugin-react-helmet`
+
+1. Install both packages:
+
+```bash
+npm install --save gatsby-plugin-react-helmet react-helmet
+```
+
+2. Add the plugin to the `plugins` array in your `gatsby-config.js` file.
+
+```javascript
+{
+    plugins: [`gatsby-plugin-react-helmet`]
+}
+```
+
+3. Use `React Helmet` in your pages:
+
+```jsx{8-12}
+import React from "react";
+import {Helmet} from "react-helmet";
+
+class Application extends React.Component {
+  render () {
+    return (
+        <div className="application">
+            <Helmet>
+                <meta charSet="utf-8" />
+                <title>My Title</title>
+                <link rel="canonical" href="http://mysite.com/example" />
+            </Helmet>
+            ...
+        </div>
+    );
+  }
+};
+```
+
+> 💡 The above example is from the [React Helmet docs](https://github.com/nfl/react-helmet#example). Check those out for more!
+
+## Keep making it better
+
+In this section we've shown you a few Gatsby-specific tools to improve your site's performance and prepare to go live.
+
+Lighthouse is a great tool for site improvements and learning -- Continue looking through the detailed feedback it provides and keep making your site better!
+
+## That's all, folks
+
+Well, not quite. Just for this tutorial. This is just the beginning. Keep going!
+
+- Did you build something cool? Share it on Twitter, tag [#buildwithgatsby](https://twitter.com/search?q=%23buildwithgatsby), and [@mention us](https://twitter.com/gatsbyjs)!
+- Did you write a cool blog post about what you learned? Share that, too!
```

Comment:
share it on twitter with same hashtag seem like a good idea?

Reply:
I also thought that was implied here, from above (since it's on the line above, no need to show it again). Can make it explicit if you want though! Y/N?

Reply:
Y